torch/distributed/_fsdp/fully_sharded_data_parallel.py

```diff
@@ -629,6 +632,76 @@ def forward(self, *args: Any, **kwargs: Any) -> Any:
 
         return outputs
 
+    @torch.no_grad()
+    def _write_back_current_shard(self):
+        for p in self.params:
+            if not p._is_sharded:  # type: ignore[attr-defined]
+                pass
```

Comment:
s/pass/continue? Also I am not sure if this is covered by the tests, as I think with `pass` this can throw an error if the param is not sharded since we attempt to access `_full_param_padded` which is not set.